src/handlebars/support.handlebars

```diff
@@ -59,6 +59,360 @@
         and we will work with you and the appropriate development team to resolve it.
         Any fixes we identify will be delivered as part of the next AdoptOpenJDK release.
       </p>
+	  
+		<h3 id="openjdk8_platforms">OpenJDK 8</h3>
+		<p>Where available, OpenJDK 8 binaries are supported on the minimum operating system levels shown in the following tables:</p>
+
+		<table border="1">
```

Comment:
Please use SCSS / CSS to define border in all places used

Reply:
Done.